package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Genotet",
   "main": "genotet.js",
   "dependencies": {
-    "bower": "^1.5.2"
+    "assert": "^1.3.0",
```

Comment:
Are we using assert and mongodb for root level package? Aren't they for server and should be declared instead in server/package.json?